### ISSUE TITLE
我在基于bert训练CNN或者其他CNNbase的模型时候，都会出现loss.backward()之后，程序停止运行的情况。

### ISSUE THREAD

**Issue by FOXaaFOX** [hscspring/Multi-Label-Text-Classification-for-Chinese#1]
如题
在原工程lonePatient下我也提了issue，这个问题至今没有解决
CNNbase都会遇到此种问题，无论是我自己写的还是复制您的代码，都会有同样的问题
RNNbase没有。
请问这是什么原因呢？没有报错提醒，我debug的时候看了，在计算loss时，tensor size也没有问题

**Comment by FOXaaFOX** [comment]
问题已经解决 应该是pytorch在不同os的问题，我从win10切换到linux 运行正常
